src/api/addresses.ts

```diff
@@ -14,37 +16,57 @@ export const fetchAddresses = async (
     client
       .getAddresses({
         startPath: DEFAULT_ETH_DERIVATION,
-        n: 10,
+        n: MAX_ADDR,
         ...overrides,
       })
       .then((addrs) => addrs.map((addr) => `${addr}`)),
   );
 };
 
-export const fetchBtcLegacyAddresses = async (n = 10): Promise<string[]> => {
+export const fetchBtcLegacyAddresses = async (
+  n = MAX_ADDR,
+  startPathIndex?: number,
```

Comment:
I suppose this could be a type since it's used 3 times, but not a big deal either way